src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -67,6 +79,8 @@ export interface ICodeActionMenuItem {
 	isSeparator: boolean;
 	isEnabled: boolean;
 	isDocumentation: boolean;
+	isHeader: boolean;
```

Comment:
Instead of adding more properties here, what if you had different types of `ICodeActionMenuItem` such as `ICodeActionHeaderMenuItem` and `ICodeActionSeparatorMenuItem`